Modules/_datetimemodule.c

```diff
@@ -4012,8 +4012,8 @@ static PyGetSetDef time_getset[] = {
  * Constructors.
  */
 
-static char *time_kws[] = {"hour", "minute", "second", "microsecond",
-                           "tzinfo", "fold", NULL};
+static const char *time_kws[] = {"hour", "minute", "second", "microsecond",
+                                "tzinfo", "fold", NULL};
```

Comment:
I think this got mis-aligned.

```suggestion
                                 "tzinfo", "fold", NULL};
```